CS1525: Invalid expression term '='

## Environments (please complete the following information):

 - UniVRM version: `0.81.0`
 - Unity version: `Unity-2019.4`
 - OS: `Windows 10`

## Describe the bug

Assets\UniGLTF\Runtime\UniGLTF\IO\TextureIO\GltfTextureImporter.cs(154,31): error CS1525: Invalid expression term '='
 Happens when Import UniVRM, doesnt show up at top

--- a/Assets/UniGLTF/Runtime/UniGLTF/IO/TextureIO/GltfTextureImporter.cs
+++ b/Assets/UniGLTF/Runtime/UniGLTF/IO/TextureIO/GltfTextureImporter.cs
@@ -151,7 +151,10 @@ namespace UniGLTF
                     {
                         name = TextureImportName.GetUnityObjectName(TextureImportTypes.StandardMap, gltfTexture.name, gltfImage.uri);
                     }
-                    sampler ??= TextureSamplerUtil.CreateSampler(data.GLTF, occlusionTextureIndex.Value);
+                    if (sampler == null)
+                    {
+                        sampler = TextureSamplerUtil.CreateSampler(data.GLTF, occlusionTextureIndex.Value);
+                    }
                     getOcclusionAsync = () => Task.FromResult(GetImageBytesFromImageIndex(data, imageIndex.Value));
                 }
             }

